src/cleo/commands/completions_command.py

```diff
@@ -37,7 +37,7 @@ class CompletionsCommand(Command):
         )
     ]
 
-    SUPPORTED_SHELLS = ("bash", "zsh", "fish")
+    SUPPORTED_SHELLS = ("bash", "zsh", "fish", "PowerShell")
```

Comment:
Would we want to recognize `ps`/`pwsh` as forms of `PowerShell`?

Additionally, would we want to ignore case for this? 
```
> poetry completions powershell

[shell] argument must be one of bash, zsh, fish, PowerShell
```